eks-distro-base/Makefile

```diff
@@ -39,6 +39,8 @@ images:
 		--local dockerfile=./ \
 		--local context=. \
 		--output type=image,oci-mediatypes=true,name=$(IMAGE),push=true
+	cd ../../eks-distro-pr-bot/eks-distro-build-tooling
```

Comment:
Can we use root of this workspace or Makefile instead of `../../`like in other files? Here is an example: https://github.com/aws/eks-distro-build-tooling/blob/main/projects/gomods/athens/upload-athens-artifacts.sh#L24

Reply:
Sure, will change it